views/includes/page-templates/important-banner.html

```diff
@@ -0,0 +1,10 @@
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+{% set html %}
+
+You need to provide some information about the overseas entity from the pre-registration period <br></br>
+{% include "includes/page-templates/relevant-period.html" %}
```

Comment:
could we enclose this text in a `<p>` element. You wont need to use `<br>` in that case also